client/src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Footer() {
+    return (
+        <div className='text-center text-lg-start text-muted bg-light'>
+            <div className='text-center p-4 mt-5 border-bottom' style={{ backgroundColor: 'rgba(0, 0, 0, 0.05)' }}>
+                © 2023 Copyright: myLearn
```

Comment:
Use the HTML entity to create a copyright symbol. Also, capitalize the first letter of the brand name.